critiquebrainz/frontend/external/musicbrainz_db/tests/__init__.py

```diff
@@ -0,0 +1,14 @@
+from brainzutils import cache
+
+
+REDIS_HOST = 'critiquebrainz_redis'
+REDIS_PORT = 6379
+REDIS_NAMESPACE = 'CB'
```

Comment:
Hmm, this is not ideal. It would be good to get these values from the test configuration instead.